### PR TITLE
OCT-1684: Implement "Calculating uniqueness" modal : client

### PR DESCRIPTION
**Pull request by jmikolajczyk**
INFO
Check after merging [PR-253](https://github.com/golemfoundation/octant/pull/253)

## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/Settings/ModalSettingsCalculatingYourUniqueness/ModalSettingsCalculatingYourUniqueness.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  const { isDesktop } = useMediaQuery();
+  const { t } = useTranslation('translation', { keyPrefix: 'views.settings' });
+
+  const [touchStart, setTouchStart] = useState<number | null>(null);
+
+  const steps = [
+    <Trans
+      className={styles.onboardingModalText}
+      i18nKey="views.settings.calculatingYourUniquenessStep1"
+      // TODO: add "Gitcoin Passport" href
```
Comment (aziolek) [inline client/src/components/Settings/ModalSettingsCalculatingYourUniqueness/ModalSettingsCalculatingYourUniqueness.tsx:28]: TODO left.

File: client/src/components/Settings/ModalSettingsCalculatingYourUniqueness/ModalSettingsCalculatingYourUniqueness.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    />,
+    <Trans
+      className={styles.onboardingModalText}
+      i18nKey="views.settings.calculatingYourUniquenessStep2"
+    />,
+    <Trans
+      className={styles.onboardingModalText}
+      components={[
+        <Button className={styles.link} href={DISCORD_LINK} variant="link3" />,
+        // TODO: "scoring 20 for humans"
```
Comment (aziolek) [inline client/src/components/Settings/ModalSettingsCalculatingYourUniqueness/ModalSettingsCalculatingYourUniqueness.tsx:39]: What is this?

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.root {
```
Comment (aziolek) [inline client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.module.scss:1]: Should use 2-space indent.

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+import Identicon from 'components/ui/Identicon';
+
+import styles from './SettingsUniquenessScoreAddresses.module.scss';
+
+const addresses = [
+  '0xe5e11cc5fb894eF5A9D7Da768cFb17066b9d35D7',
+  '0xDA4069005073E91301DcC38797d7159B301a3590',
+  '0x2043c7262A19493cD293d865c56482AF122f8882',
+];
```
Comment (aziolek) [inline client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx:12]: What's that?

**Review by aziolek — CHANGES_REQUESTED** [review]

File: client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx (hunk 4 above)
Comment (jmikolajczyk) [reply client/src/components/Settings/SettingsUniquenessScoreAddresses/SettingsUniquenessScoreAddresses.tsx:12]: mocked addresses -> It's UI only task

**Pull request merged**